packages/utils/test/crypto/ec-utils.test.ts

```diff
@@ -27,11 +27,10 @@ describe('Utils/EcUtils', () => {
   describe('getAddressFromPrivateKey', () => {
     it('can get Address From PrivateKey', () => {
       const identity = getAddressFromPrivateKey(rawId.privateKey);
-      // 'getAddressFromPrivateKey() error'
       expect(identity).toBe(rawId.address);
     });
     it('cannot get Address From PrivateKey if the private key is wrong', () => {
-      // 'getAddressFromPrivateKey() error'
+      // getAddressFromPrivateKey() error
```

Comment:
In case of failure tests, I kept the comments but removed the quotes

Reply:
these comments were generated when migrating from mocha to jest, they can totally be removed

Reply:
OK then, I'm removing them altogether